way_to_home/static/src/components/loginForm/loginForm.js

```diff
@@ -112,17 +120,19 @@ class LoginForm extends Component {
             this.setState({disable_button: false});
     };
 
-    setError = (error) => {
-        this.setState({ajax_error: error});
-    };
-
-    changeSaveCookies = () => {
-        this.setState({remember_me: !this.state.remember_me});
+    handleSigninGoogle = () => {
+        axios.get('/api/v1/user/auth_via_google')
+            .then((response ) => {
+                window.location.replace(response.data.url);
+            })
+            .catch((error) => {
+                self.setError(error.response.data);
```

Comment:
Translate this error message to Ukrainian.

Reply:
How?